src/lib/pickles/test/optional_custom_gates/pickles_test_optional_custom_gates.ml

```diff
@@ -192,6 +197,48 @@ let main_foreign_field_mul () =
   add_plonk_constraint
     (Raw { kind = Zero; values = [| fresh_int 0 |]; coeffs = [||] })
 
+(* Parameters *)
+let random_table_id = Random.State.int state 1_000
+
+let size = 1 + Random.State.int state 1_000
+
+let values =
+  Array.init size ~f:(fun _ ->
+      let x = Random.State.int state 1_000_000 in
+      Field.Constant.of_int x )
+
+let idx1 = Random.State.int state size
+
+let idx2 = Random.State.int state size
+
+let idx3 = Random.State.int state size
+
+let main_fixed_lookup_tables () =
+  let table_id = random_table_id in
+  let size = size in
+  let indexes = Array.init size ~f:Field.Constant.of_int in
+  let values = values in
```

Comment:
Same thing here: do we actually need the `let x = x in `?

Reply:
Same than https://github.com/MinaProtocol/mina/pull/14052#discussion_r1318761692